code/datums/mind.dm

```diff
@@ -29,6 +29,10 @@
 
 */
 
+#define MIND_MURDERBONE_GENERAL (1<<0)
+#define MIND_MURDERBONE_HIJACK (1<<1)
+#define MIND_MURDERBONE_MAATYR (1<<2)
```

Comment:
Either these should be a single thing, or general should be replaced with ALL, so that you only add 1 murderbone flag and then check if any are active

Reply:
removed now